### ISSUE TITLE
Zarf support for EKS

### ISSUE THREAD

**Issue by RothAndrew** [defenseunicorns/zarf#264]
**Story:**
As a user of EKS and Zarf, I want the ability to deploy Zarf packages to EKS, so that I can use Zarf in my environment.

**Acceptance Criteria:**
- The Games example deploys successfully to EKS (Commercial) (definition of "successfully" is that all expected pods are running)

**Comment by RothAndrew** [comment]
Potential future backlog items:
- EKS (GovCloud)
- Big Bang example deploys to EKS (Commercial then GovCloud)
- Software Factory package (a.k.a. DI2-ME) deploys to EKS (Commercial then GovCloud)

**Comment by RothAndrew** [comment]
Blocked by #237

**Comment by RothAndrew** [comment]
@team, looking for engineering feedback on this before it gets moved to Ready to Start

* Is the outcome and acceptance criteria well understood?
* Is it reasonable?
* Can the backlog item be made smaller without arbitrarily slicing it into little valueless chunks?

**Comment by RothAndrew** [comment]
Adding to the "DI2-ME" milestone, since it is needed for that work to truly be valuable, but this backlog item is also very valuable outside of the context of the "DI2-ME" package.

**Comment by RothAndrew** [comment]
Blocked by #335

**Comment by RothAndrew** [comment]
This shouldn't be closed by #329 actually since it needs an E2E test that deploys to EKS

**Comment by RothAndrew** [comment]
@jeff-mccoy we need to reopen this since the description calls for an E2E test, unless we want to make a new issue for that.

**Comment by jeff-mccoy** [comment]
Closing this one, updated AC to remove the E2E for this one as we've been working/testing on EKS for a couple of months now.
